site/layouts/partials/docs-sidebar.html

```diff
@@ -16,26 +16,27 @@
     {{- $is_active_group := eq $.Page.Params.group $group_slug -}}
 
     {{- if $group.pages }}
-      <li class="mb-1">
-        <button class="btn d-inline-flex align-items-center rounded{{ if not $is_active_group }} collapsed{{ end }}" data-bs-toggle="collapse" data-bs-target="#{{ $group_slug }}-collapse" aria-expanded="{{ $is_active_group }}"{{ if $is_active_group }} aria-current="true"{{ end }}>
+      <li class="bd-links-group py-2">
+        <h6 class="bd-links-heading d-flex w-100 align-items-center">
```

Comment:
`h6` feels weird here, the headings hierarchy is quite messy: 
![image](https://user-images.githubusercontent.com/2890570/154106707-e4bf4242-970d-4e8c-8efa-bc12441d94ff.png)
Maybe use a `<p class="h6">` instead?
@patrickhlauke Is it important or not that much? I remember your "When headings fails WCAG" post.